### PR TITLE
front-end implementation of login/register page completed

### PR DESCRIPTION
**Pull request by Archdragon79**
- Fixed borked design of Login and Register pages
- Fixed widget hierarchies
- Added transitions
- Separated login and home trees

### CODE REVIEW

File: lib/home.dart
Comment (Mast3rwaf1z) [inline lib/home.dart:1]: was this meant for another pr?

File: lib/login.dart [hunk 1]
```diff
@@ -1,138 +1,84 @@
...
+}
+
+class LoginScreenState extends State<LoginScreen> {
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
         appBar: AppBar(
-          backgroundColor: const Color.fromARGB(179, 85, 85, 85),
+          backgroundColor: const Color.fromARGB(255, 30, 30, 30),
```
Comment (Mast3rwaf1z) [inline lib/login.dart:21]: can't you do any form of global styling? like have a default color palette for the entire frontend?

**Review by Mast3rwaf1z — CHANGES_REQUESTED** [review]
some minor things

File: lib/home.dart
Comment (Archdragon79) [reply lib/home.dart:1]: no, else the nav breaks

File: lib/login.dart (hunk 1 above)
Comment (Archdragon79) [reply lib/login.dart]: yes, see new changes

**Review by Mast3rwaf1z — APPROVED** [review]
nice!

**Pull request merged**